src/components/Categories.js

```diff
@@ -1,7 +1,16 @@
-import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { CheckStatus } from '../redux/categories/categories';
 
-const Categories = () => (
-  <button className="check-status" type="button">Check Status</button>
-);
+const Categories = () => {
+  const dispatch = useDispatch();
+  const categories = useSelector((state) => state.categories);
+
+  return (
+    <>
+      <h1>{categories}</h1>
+      <button onClick={() => dispatch(CheckStatus())} className="check-status" type="button">Check Status</button>
```

Comment:
- [ ] When I click the Check Status button, it is supposed to show the text "Under Construction", but it's not working presently.  Kindly work on that👍